packages/client/src/DisplayContent.tsx

```diff
@@ -17,71 +19,77 @@ const useStyles = makeStyles(_theme => ({
   }
 }))
 
-type Props = graphql.Content & { className?: string }
+type Props = graphql.Content & { accountId: string; className?: string }
 
 export default function DisplayContent({
+  accountId,
   type,
   subtype,
   content,
   className
 }: Props) {
   const classes = useStyles()
-  if (type === "text" && subtype === "html") {
-    return displayHtml(content, clsx(className, "html-content", classes.body))
-  } else if (type === "text" && subtype === "markdown") {
-    return displayMarkdown(
-      content,
-      clsx(className, "markdown-content", classes.body)
-    )
-  } else if (type === "text") {
-    return displayText(
-      content,
-      clsx(className, "text-content", clsx(classes.body, classes.textContent))
-    )
-  } else {
-    return displayUnknown(
-      { type, subtype, content },
-      clsx(className, classes.body)
-    )
-  }
+  return (
+    <Location>
+      {({ navigate }) => {
```

Comment:
`Location` is a higher-order component provided by `@reach/router` that provides access to routing objects from context such as the `navigate` function.